docs/getting-started/cluster-class/provision.md

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 1
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Create a workload cluster
+
+This section will guide you through creating a `ClusterClass` which you will then use to provision a workload `Cluster`. The Cluster API book includes a [ClusterClass section](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-class/) with detailed information on what is supported and how you can use the powerful abstraction this feature provides, including operations such as [patching class values](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-class/write-clusterclass#clusterclass-with-patches) and [operating a managed cluster](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-class/operate-cluster). We recommend you familiarize with the feature to get the most out of it.
+
+:::note
+This guide uses the [examples repository](https://github.com/rancher-sandbox/rancher-turtles-fleet-example/tree/templates).
+:::
+
+## Providers guide
+
+<Tabs>
+  <TabItem value="azure-aks" label="CAPZ AKS" default>
+  ## Prerequisites
+    - Rancher Manager cluster with Rancher Turtles installed
+    - Configure cloud credentials for Azure in Rancher: `Cluster Management` > `Cloud Credentials`.
+        - Keep the name you assign to the new set of credentials.
+    - Install the [CAPI Infrastructure Provider for Azure](https://github.com/kubernetes-sigs/cluster-api-provider-azure/) using the [`CAPIProvider` resource](../../tasks/capi-operator/basic_cluster_api_provider_installation.md).
+        ```yaml
+        apiVersion: v1
+        kind: Namespace
+        metadata:
+        name: capz-system
+        ---
+        apiVersion: turtles-capi.cattle.io/v1alpha1
+        kind: CAPIProvider
+        metadata:
+        name: azure
+        namespace: capz-system
+        spec:
+        type: infrastructure
+        name: azure
+        credentials:
+            rancherCloudCredential: <rancher-credential-name> # Rancher credentials secret for Azure
+        configSecret:
+            name: azure-variables
+        variables:
+            CLUSTER_TOPOLOGY: "true"
+            EXP_CLUSTER_RESOURCE_SET: "true"
+            EXP_MACHINE_POOL: "true"
+            EXP_AKS_RESOURCE_HEALTH: "true"
```

Comment:
This CAPIProvider should look like
```yaml

        apiVersion: turtles-capi.cattle.io/v1alpha1
        kind: CAPIProvider
        metadata:
            name: azure
            namespace: capz-system
        spec:
            type: infrastructure
```
in order to work. All other fields are not required in AKS case.